### PR TITLE
[docs] [integrations] Integrations page for Django with YugabyteDB

### LINKED ISSUE #9382: [docs] Document Django in Integrations
CC: @kneeraj

### PR DESCRIPTION
**Pull request by aishwarya24**
Resolves #9382 

- Verified the steps and modified parts of the document to make it compatible for Django 4.0
- Created a document for integrations section.

@netlify /latest/integrations/django-rest-framework/

### CODE REVIEW

File: docs/content/latest/integrations/django-rest-framework.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+python3 manage.py makemigrations
+```
+
+- Migrate the changes to the database with the command:
+
+```python
+python3 manage.py migrate
+```
+
+- A users table should be created in your database. Verify that the table is created with the [ysqlsh](../../explore/ysql-language-features/databases-schemas-tables/#list-tables) client shell.
```
Comment (lizayugabyte) [inline docs/content/latest/integrations/django-rest-framework.md:170]: Suggestion:

Use the [ysqlsh](../../explore/ysql-language-features/databases-schemas-tables/#list-tables) client shell to verify that the users table has been created in your database.

File: docs/content/latest/integrations/django-rest-framework.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+## Run the application
+
+- Run the django project with the command:
```
Comment (lizayugabyte) [inline docs/content/latest/integrations/django-rest-framework.md:174]: Intro sentence before the list is missing

File: docs/content/latest/integrations/django-rest-framework.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+```python
+python3 manage.py runserver 8080
+```
+
+- Insert a row with the command:
```
Comment (lizayugabyte) [inline docs/content/latest/integrations/django-rest-framework.md:180]: the following command:

File: docs/content/latest/integrations/django-rest-framework.md (hunk 2 above)
Comment (lizayugabyte) [inline docs/content/latest/integrations/django-rest-framework.md:174]: the following command:

File: docs/content/latest/integrations/django-rest-framework.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+```sh
+$ curl --data '{ "firstName" : "John", "lastName" : "Smith", "email" : "jsmith@yb.com" }' \
+       -v -X POST -H 'Content-Type:application/json' http://localhost:8080/users
+```
+
+- Verify that the new row is inserted with the command:
```
Comment (lizayugabyte) [inline docs/content/latest/integrations/django-rest-framework.md:187]: Suggestion: 
Verify that the new row has been inserted by executing the following command:

**Review by lizayugabyte — CHANGES_REQUESTED** [review]
Made some comments

File: docs/content/latest/integrations/django-rest-framework.md [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+from testdb.views import UserViewSet
+
+router = routers.SimpleRouter(trailing_slash=False)
+router.register(r'users', UserViewSet)
+urlpatterns = [
+    re_path(r'^', include(router.urls))
+]
+```
+
+For Django versions below 4.0 add the following code in `urls.py` instead, since you can import the urls using `django.conf.urls`:
```
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/latest/integrations/django-rest-framework.md:110]: Thank you for this one ,will remember this rule.

File: docs/content/latest/integrations/django-rest-framework.md [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+    parent: integrations
+    weight: 670
+isTocNested: true
+showAsideToc: true
+---
+
+This document describes how to use [Django REST framework](https://www.django-rest-framework.org/), an ORM library in Python with YugabyteDB.
+
+## Prerequisites
+
```
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/latest/integrations/django-rest-framework.md:17]: @Liza I have asked regarding what the minimum YugabyteDB version must be and will include when I have that detail. Regarding Django for backend version- I see one repo with instructions on installing it. There is no version mentioned : https://github.com/yugabyte/yb-django . So I think we can leave it as it is. I have modified the rest accordingly .

**Review by lizayugabyte — APPROVED** [review]
LGTM

**Review by Sfurti-yb — APPROVED** [review]
LGTM

**Pull request merged**